invoices/invoiceregistry_test.go

```diff
@@ -46,19 +50,27 @@ var (
 	testHodlInvoice = &channeldb.Invoice{
 		Terms: channeldb.ContractTerm{
 			PaymentPreimage: channeldb.UnknownPreimage,
-			Value:           lnwire.MilliSatoshi(100000),
+			Value:           testInvoiceAmt,
 			Features:        testFeatures,
 		},
 	}
 )
 
-func newTestContext(t *testing.T) (*InvoiceRegistry, func()) {
+type testContext struct {
+	registry *InvoiceRegistry
+
+	cleanup func()
+	t       *testing.T
+}
+
+func newTestContext(t *testing.T) *testContext {
+
```

Comment:
unnecessary newline?

Reply:
hmm, removed again. wondering if there is something weird in my rebase aliases